includes/class-freemius.php

```diff
@@ -2058,6 +2063,48 @@ function _submit_uninstall_reason_action() {
             exit;
         }
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since  2.1.4
+         */
+        function cancel_subscription_or_trial_ajax_action() {

```

Comment:
@fajardoleo we need to add some error handling to the cancellation to avoid cases when a user cancels a subscription and later get billed because of some error.